md2html.sh

```diff
@@ -16,12 +18,36 @@ while getopts "hl" opt; do
 done
 
 for file in "$@"; do
-	pandoc -f markdown_github "${file}" -t asciidoc -o "${file%.md}.tmp.adoc"
-	if [ "$convert_link" -ne 0 ]; then
-		sed -i -e "s/\.md\>/\.html/g" "${file%.md}.tmp.adoc"
-	fi
-	touch -r "${file}" "${file%.md}.tmp.adoc"
-	TZ=UTC asciidoc -o "${file%.md}.html" -a footer-style=none -a toc2 -a source-highlighter=highlight "${file%.md}.tmp.adoc"
-	tr -d '\r' < "${file%.md}.html" > "${file%.md}.tmp.adoc"
-	mv "${file%.md}.tmp.adoc" "${file%.md}.html"
+        md2html_tool=""
+        # RHEL 9 in brew cannot use pandoc, hence trying kramown first
+        if rpm -q rubygem-kramdown-parser-gfm > /dev/null; then
+          md2html_tool=kramdown
+        elif rpm -q pandoc >/dev/null; then
+          md2html_tool=pandoc
+        fi
+
+        # With kramdown, convert directly to HTML
+        if [ $md2html_tool == kramdown ]; then
+          # Run kramdown with `ruby` to specify the encoding as ruby uses US-ASCII by default
+          ruby --encoding=UTF-8 -S $md2html_tool --extension parser-gfm \
```

Comment:
Yes, e.g. run `kramdown -i GFM README.md`, this either passes with `0` rc, or fails with `Error: kramdown has no parser to handle the specified input format: GFM` and `1` rc.
I will use this instead of the `rpm -ql` check. Thank you

Reply:
As per our IRC discussion, I will use `LC_ALL=en_US.UTF-8 ruby` as you suggested. Thank you

Reply:
As per our discussion, I used `LC_ALL=C.UTF-8` with the kramdown command, this works on Fedora just fine:
https://koji.fedoraproject.org/koji/taskinfo?taskID=67821072
This will also work on RHEL 9. This won't work on RHEL 7, but RHEL 7 will use pandoc and asciidoctor, not kramdown.